docs/source/notebooks/api_quickstart.ipynb

```diff
@@ -16,12 +16,13 @@
      "name": "stdout",
```

Comment:
<p>This example would be even nicer if you use the <code>pm.Data</code>  container, to illustrate :)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc3/pull/3911/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='102'/>